patches/esp_app_format_weak_v4.3.diff

```diff
@@ -7,7 +7,7 @@ index db6a33529d..6c82cae929 100644
 
  // Application version info
 -const __attribute__((section(".rodata_desc"))) esp_app_desc_t esp_app_desc = {
-+const __attribute__((__weak__)) __attribute__((section(".rodata_desc"))) esp_app_desc_t esp_app_desc = {
++const __attribute__((weak)) __attribute__((section(".rodata_desc"))) esp_app_desc_t esp_app_desc = {
```

Comment:
Why?

Reply:
Apparently that's the correct syntax. CI doesn't compile against that version of esp-idf, so it did not catch it